doc/classes/VisualShaderNode.xml

```diff
@@ -37,16 +40,19 @@
 			<argument index="1" name="value" type="Variant">
 			</argument>
 			<description>
+				Sets the default value for the selected input [code]port[/code].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="output_port_for_preview" type="int" setter="set_output_port_for_preview" getter="get_output_port_for_preview" default="-1">
+			Sets the output port index which will be showed for preview. If set to [code]-1[/code] no port will be open for preview.
 		</member>
 	</members>
 	<signals>
 		<signal name="editor_refresh_request">
 			<description>
+				Emitted when the node requests an editor refresh. Currently called only in setter of [member VisualShaderNodeTexture.source] of [VisualShaderNodeTexture] and [VisualShaderNodeCubeMap] (and their derivatives).
```

Comment:
... of [member VisualShaderNodeTexture.source], [VisualShaderNodeTexture], and [VisualShaderNodeCubeMap] (and their derivatives).

Sorry, I missed this one before.